gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -423,9 +441,9 @@ gst_tensor_trainer_check_invalid_param (GstTensorTrainer * trainer)
   g_return_val_if_fail (trainer != NULL, FALSE);
 
   /* Parameters that can be retrieved from caps will be removed */
-  if (!trainer->fw_name || !trainer->model_config || !trainer->model_save_path
-      || trainer->prop.num_epochs <= 0 || trainer->prop.num_inputs <= 0
-      || trainer->prop.num_labels <= 0) {
+  if (!trainer->fw_name || !trainer->prop.model_config
+      || !trainer->prop.model_save_path || trainer->prop.num_epochs <= 0
```

Comment:
Is it needed to check null for trainer->prop.model_config or trainer->model_save_path?

Reply:
Check null ptr is meaningless, let's compare with the "" string.

Reply:
null check is also necessary. thank you.